src/adapters/mc/ssh/19params/LinuxSshServerSecurityBaseline_DeployIfNotExists.json

```diff
@@ -639,7 +639,7 @@
                                                 "version": "1.0.0",
                                                 "contentType": "Custom",
                                                 "contentUri": "https://github.com/Azure/azure-osconfig/releases/download/test_policy_package/LinuxSshServerSecurityBaseline.zip",
```

Comment:
Should we update the contentUri to point to `ignite_2024` instead of `test_policy_package`?